src/App.jsx

```diff
@@ -1,12 +1,12 @@
-import React, { Component } from 'react';
```

Comment:
Why did you remove the spaces?

Reply:
Are you using a linter?

Reply:
Yes I am using a linter, I need to add the rc to the project oops